python_modules/dagster/dagster/_core/definitions/module_loaders/utils.py

```diff
@@ -1,19 +1,49 @@
 import pkgutil
 from importlib import import_module
 from types import ModuleType
-from typing import Iterable, Iterator, Mapping, Tuple, Type, Union
+from typing import Any, Iterable, Iterator, Mapping, Tuple, Type, Union
 
 from dagster._core.definitions.asset_key import AssetCheckKey, AssetKey
 from dagster._core.definitions.asset_spec import AssetSpec
 from dagster._core.definitions.assets import AssetsDefinition
 from dagster._core.definitions.cacheable_assets import CacheableAssetsDefinition
+from dagster._core.definitions.executor_definition import ExecutorDefinition
+from dagster._core.definitions.job_definition import JobDefinition
+from dagster._core.definitions.logger_definition import LoggerDefinition
+from dagster._core.definitions.partitioned_schedule import (
+    UnresolvedPartitionedAssetScheduleDefinition,
+)
+from dagster._core.definitions.schedule_definition import ScheduleDefinition
+from dagster._core.definitions.sensor_definition import SensorDefinition
 from dagster._core.definitions.source_asset import SourceAsset
+from dagster._core.definitions.unresolved_asset_job_definition import UnresolvedAssetJobDefinition
+from dagster._core.executor.base import Executor
 
 LoadableAssetObject = Union[AssetsDefinition, AssetSpec, SourceAsset, CacheableAssetsDefinition]
-LoadableDagsterObject = LoadableAssetObject  # For now
+ScheduleDefinitionObject = Union[ScheduleDefinition, UnresolvedPartitionedAssetScheduleDefinition]
+JobDefinitionObject = Union[JobDefinition, UnresolvedAssetJobDefinition]
```

Comment:
maybe:

LoadableAssetDef
LoadableScheduleDef
LoadableJobDef

?

Reply:
LoadableAssetDef I think is particularly confusing given that it can be a SourceAsset or AssetSpec, for example